swagger/voice_calls/voice_calls.json

```diff
@@ -0,0 +1,733 @@
+                      "properties": {
+                        "data": { "type": "object", "$ref": "#/components/schemas/VoiceCallResponseDto" }
+                      }
+                    }
+                  ]
+                }
+              }
+            }
+          }
+        },
+        "tags": ["voice-calls"],
+        "security": [{ "Bearer": [] }]
+      }
+    },
+    "/v3/communication/voice_calls/recordings/{uid}": {
+      "delete": {
+        "operationId": "VoiceCallsController_deleteVoiceCallRecording",
+        "parameters": [{ "name": "uid", "required": true, "in": "path", "schema": { "type": "string" } }],
+        "responses": {
+          "200": {
+            "description": "",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "allOf": [
+                    { "$ref": "#/components/schemas/Response" },
+                    {
+                      "properties": {
+                        "data": { "type": "object", "$ref": "#/components/schemas/VoiceCallResponseDto" }
+                      }
+                    }
+                  ]
+                }
+              }
+            }
+          }
+        },
+        "tags": ["voice-calls"],
+        "security": [{ "Bearer": [] }]
+      }
+    },
+    "/v3/communication/quotas": {
+      "post": {
+        "operationId": "QuotasController_create",
+        "parameters": [],
+        "requestBody": {
+          "required": true,
+          "content": { "application/json": { "schema": { "$ref": "#/components/schemas/CreateQuota" } } }
+        },
+        "responses": {
+          "201": {
+            "description": "",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "allOf": [
+                    { "$ref": "#/components/schemas/Response" },
+                    {
+                      "properties": {
+                        "data": { "type": "object", "$ref": "#/components/schemas/QuotaResponseDto" }
+                      }
+                    }
+                  ]
+                }
+              }
+            }
+          }
+        },
+        "tags": ["quotas"]
+      },
+      "get": {
+        "operationId": "QuotasController_getQuota",
+        "parameters": [],
+        "responses": {
+          "200": {
+            "description": "",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "allOf": [
+                    { "$ref": "#/components/schemas/Response" },
+                    {
+                      "properties": {
+                        "data": { "type": "object", "$ref": "#/components/schemas/QuotaResponseDto" }
+                      }
+                    }
+                  ]
+                }
+              }
+            }
+          }
+        },
+        "tags": ["quotas"]
+      }
+    },
+    "/v3/communication/quotas/current": {
```

Comment:
Why do we need this end point? you can add a filter to the list with a parameter "is_current"? we do not want these "special case" end points if they are not absolutely needed

Reply:
@nikitarablus @goldi-tz